book/run/optimism.md

```diff
@@ -0,0 +1,118 @@
+# Running Reth on OP Stack chains
+
+`reth` ships with the `optimism` feature flag, enabling support for OP Stack chains out of the box. Optimism has a small diff from the [L1 EELS][l1-el-spec], comprising of the following key changes:
+1. A new transaction type, [`0x7E (Deposit)`][deposit-spec], which is used to deposit funds from L1 to L2.
+1. Modifications to the `PayloadAttributes` that allow the [sequencer][sequencer] to submit transactions in-order to the EL through the Engine API.
```

Comment:
`in-order` reads slightly ambiguous to me, can we say `first-in first-out (FIFO)` instead or something to that effect?

Reply:
Agree - exhaustively describing the behavior of sequencer tx ordering is a bit verbose for here, though, imo. What do you think about the update?

Reply:
Discussed offline, once updated gtg 👍